### DISCUSSION TITLE
How can I change my Birdnet password ?

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#732] [Q&A] [answered]
Hi folks,
This may be an easy one for you, but how can I change my Birdnet-Pi password ?
Can it be done from my computer using Linux terminal and ssh ?
Thanks !

**Comment by JohnJoosten** [comment, marked as answer]
From Birdnet interface use Tools, Webterminal and login to your webterminal on your PI
type "sudo Raspi-Config"
select 1 System Options
select S3 Password
You will asked to enter a new password twice.
press "finish" to leave

**Reply by Fish31520** [reply]
@JohnJoosten 
Perfect ! Thanks !

**Reply by ehpersonal38** [reply]
Yes these are good instructions for changing the password to your Raspberry Pi (used for SSHing into it). It's unclear from the original post if he wants to do this, or change his BirdNET-Pi web page password.

To change your BirdNET-Pi web page password:
1. On the web page, go to Tools -> Settings -> Advanced Settings
2. Find the input box for password, and change it
3. Click save at the bottom of the page

**Reply by Fish31520** [reply]
@ehpersonal38 
I've got to admit that my question was far from clear ! 😅
In fact I wanted to know how to change the password used to connect when I'm using ```ssh -X user@birdnet.local``` on my Linux computer.
